Plugins/ScenarioLoader/Plugin.cs

```diff
@@ -37,9 +37,14 @@ public bool CheckIntercept(string sender, string message)
             return "A scenario is already running.";
 
         string path = string.Join(' ', message.ToLower().Trim().Split()[1..]);
-        executingTask = Task.Run(async () => await ExecuteFileAsync(server, path));
+        if (!File.Exists(path))
+            return $"File {path} not found.";
+        else
+        {
+            executingTask = Task.Run(async () => await ExecuteFileAsync(server, path));
```

Comment:
```suggestion
            executingTask = ExecuteFileAsync(server, path);
```

This'll help with exception propagation.

Reply:
Change accepted for better exception propagation.